lib/rules/no-unused-prop-types.js

```diff
@@ -897,6 +897,13 @@ module.exports = {
         }
       },
 
+      JSXSpreadAttribute: function(node) {
+        const component = components.get(utils.getParentComponent());
```

Comment:
It seems super weird to me that `getParentComponent` infers the `node` rather than taking it as an argument (not your code ofc, just thought I'd comment)